cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -211,6 +215,17 @@ func main() {
 	if memprofile != "" {
 		cfg.MemProfile = memprofile
 	}
+	if netprofile {
+		cfg.NetProfile = netprofile
+	}
+
+	if cfg.NetProfile {
+		cfg.Log.Level = "debug"
```

Comment:
we shouldn't do this if level is trace, should us? ☝️

Reply:
Is this actually required? I would vote for keeping the same functionality: if `NetProfile` is enabled you get the logs according to the configured log_level. If you want more logs, you enable debug/trace from the config file.
In my opinion would be confusing that you explicitly set a log level and that gets overwritten by another configuration option that is not actually related with the logger configuration. wdyt?